test/e2e/storage/testsuites/ephemeral.go

```diff
@@ -467,7 +470,9 @@ func VolumeSourceEnabled(c clientset.Interface, t *framework.TimeoutContext, ns
 		// "Invalid" because it uses a feature that isn't supported.
 		return false, nil
 	default:
-		// Unexpected error.
-		return false, err
+		// Unexpected error. This may be due to tests with node skew, so the error is
```

Comment:
I'm not sure we've actually supported this kind of node skew in our e2e tests. I think even for things like upgrade tests from version X to X+1, we run e2e version X.

I think also using the apiserver to check if a feature is enabled is an antipattern in our tests. Normally we add a feature tag to a test and then remove that feature tag when a feature goes beta and is on by default. By skipping the test, then we potentially miss apiserver bugs for this feature because we wouldn't get a notification of a test failure.

Reply:
Fair enough. So really VolumeSourceEnabled should just be removed and feature tags should be used instead. Looks like it's just used in ephemeral.go anyway.